### ISSUE TITLE
godoctor causing dockerfile build failure

### ISSUE THREAD

**Issue by MarkDHarris** [godoctor/godoctor#55]
I am using "go get -v -u github.com/godoctor/godoctor" in a dockerfile and getting a docker build failure at :

```
#18 22.56 go: downloading github.com/godoctor/godoctor v0.0.0-20200702010311-8433dcb3dc61
#18 22.92 go: downloading golang.org/x/tools v0.0.0-20191028215554-80f3f9ca0853
#18 22.92 go: downloading github.com/willf/bitset v1.1.10
#18 23.05 go: downloading github.com/willf/bitset v1.2.0
#18 23.54 go get: github.com/willf/bitset@v1.1.10 updating to
#18 23.54       github.com/willf/bitset@v1.2.0: parsing go.mod:
#18 23.54       module declares its path as: github.com/bits-and-blooms/bitset
#18 23.54               but was required as: github.com/willf/bitset
```

Is there a fix or mitigation?

Dockerfile

```
RUN go get -v -u github.com/go-delve/delve/cmd/dlv \
    && go get -v -u github.com/mdempsky/gocode \
    && go get -v -u github.com/uudashr/gopkgs/v2/cmd/gopkgs \
    && go get -v -u github.com/ramya-rao-a/go-outline \
    && go get -v -u github.com/acroca/go-symbols \
    && go get -v -u golang.org/x/tools/cmd/guru \
    && go get -v -u golang.org/x/tools/cmd/gorename \
    && go get -v -u github.com/cweill/gotests/... \
    && go get -v -u github.com/fatih/gomodifytags \
    && go get -v -u github.com/josharian/impl \
    && go get -v -u github.com/davidrjenni/reftools/cmd/fillstruct \
    && go get -v -u github.com/haya14busa/goplay/cmd/goplay \
    && go get -v -u github.com/godoctor/godoctor \    
    && go get -v -u github.com/rogpeppe/godef \
    && go get -v -u github.com/sqs/goreturns \
    && go get -v -u golang.org/x/lint/golint \
    && go get -v -u github.com/zmb3/gogetdoc \    
    && go get -v -u honnef.co/go/tools/... \
    && go get -v -u golang.org/x/tools/cmd/goimports \
    && go get -v -u github.com/mgechev/revive
```

Logs:

```
#18 [14/17] RUN go get -v -u github.com/go-delve/delve/cmd/dlv     && go get -v -u github.com/mdempsky/gocode     && go get -v -u github.com/uudashr/gopkgs/v2/cmd/gopkgs     && go get -v -u github.com/ramya-rao-a/go-outline     && go get -v -u github.com/acroca/go-symbols     && go get -v -u golang.org/x/tools/cmd/guru     && go get -v -u golang.org/x/tools/cmd/gorename     && go get -v -u github.com/cweill/gotests/...     && go get -v -u github.com/fatih/gomodifytags     && go get -v -u github.com/josharian/impl     && go get -v -u github.com/davidrjenni/reftools/cmd/fillstruct     && go get -v -u github.com/haya14busa/goplay/cmd/goplay     && go get -v -u github.com/godoctor/godoctor     && go get -v -u github.com/rogpeppe/godef     && go get -v -u github.com/sqs/goreturns     && go get -v -u golang.org/x/lint/golint     && go get -v -u github.com/zmb3/gogetdoc     && go get -v -u honnef.co/go/tools/...     && go get -v -u golang.org/x/tools/cmd/goimports     && go get -v -u github.com/mgechev/revive
#18 sha256:5c0c506a58e2bcebe5aa151efc4dce315351188fc96ebccda965e3531a522c5f
#18 0.636 go: downloading github.com/go-delve/delve v1.6.1
#18 2.233 go: downloading github.com/sirupsen/logrus v1.6.0
#18 2.233 go: downloading github.com/spf13/cobra v0.0.0-20170417170307-b6cb39589372
#18 2.236 go: downloading github.com/mattn/go-isatty v0.0.3
#18 2.247 go: downloading gopkg.in/yaml.v2 v2.2.1
#18 2.258 go: downloading github.com/cosiner/argv v0.1.0
#18 2.260 go: downloading github.com/mattn/go-colorable v0.0.0-20170327083344-ded68f7a9561
#18 2.267 go: downloading github.com/peterh/liner v0.0.0-20170317030525-88609521dc4b
#18 2.338 go: downloading github.com/google/go-dap v0.5.0
#18 2.347 go: downloading github.com/mattn/go-isatty v0.0.13
#18 2.368 go: downloading github.com/sirupsen/logrus v1.8.1
#18 2.379 go: downloading github.com/mattn/go-colorable v0.1.8
#18 2.385 go: downloading golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
#18 2.386 go: downloading github.com/spf13/cobra v1.1.3
#18 2.388 go: downloading gopkg.in/yaml.v2 v2.4.0
#18 2.391 go: downloading github.com/peterh/liner v1.2.1
#18 2.405 go: downloading go.starlark.net v0.0.0-20200821142938-949cc6f4b097
#18 2.434 go: downloading github.com/hashicorp/golang-lru v0.5.4
#18 2.439 go: downloading golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4
#18 2.549 go: downloading github.com/konsorten/go-windows-terminal-sequences v1.0.3
#18 2.618 go: downloading github.com/cpuguy83/go-md2man v1.0.10
#18 2.692 go: downloading github.com/spf13/pflag v0.0.0-20170417173400-9e4c21054fa1
#18 2.723 go: downloading golang.org/x/arch v0.0.0-20210502124803-cbf565b21d1e
#18 2.723 go: downloading go.starlark.net v0.0.0-20210602144842-1cdb82c9e17a
#18 2.807 go: downloading golang.org/x/sys v0.0.0-20210603125802-9665404d3644
#18 2.810 go: downloading github.com/inconshreveable/mousetrap v1.0.0
#18 2.823 go: downloading github.com/russross/blackfriday v1.5.2
#18 2.828 go: downloading github.com/spf13/pflag v1.0.5
#18 2.963 go: downloading github.com/russross/blackfriday v1.6.0
#18 5.767 go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0
#18 5.796 go: downloading github.com/mattn/go-runewidth v0.0.3
#18 5.861 go: downloading github.com/russross/blackfriday/v2 v2.0.1
#18 5.902 go: downloading github.com/mattn/go-runewidth v0.0.13
#18 6.008 go: downloading github.com/russross/blackfriday/v2 v2.1.0
#18 6.043 go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
#18 6.144 go: downloading github.com/rivo/uniseg v0.2.0
#18 6.329 golang.org/x/sys/internal/unsafeheader
#18 6.331 github.com/rivo/uniseg
#18 6.341 github.com/hashicorp/golang-lru/simplelru
#18 6.344 go.starlark.net/internal/spell
#18 6.350 gopkg.in/yaml.v2
#18 6.350 golang.org/x/sys/unix
#18 6.351 github.com/go-delve/delve/pkg/goversion
#18 6.352 github.com/cosiner/argv
#18 6.357 github.com/go-delve/delve/pkg/astutil
#18 6.359 github.com/go-delve/delve/pkg/dwarf/util
#18 6.361 github.com/go-delve/delve/pkg/dwarf/regnum
#18 6.377 github.com/go-delve/delve/pkg/elfwriter
#18 6.378 github.com/go-delve/delve/pkg/version
#18 6.384 golang.org/x/arch/arm64/arm64asm
#18 6.391 golang.org/x/arch/x86/x86asm
#18 6.397 github.com/go-delve/delve/pkg/terminal/colorize
#18 6.400 github.com/go-delve/delve/pkg/dwarf/frame
#18 6.440 github.com/go-delve/delve/pkg/dwarf/op
#18 6.460 github.com/go-delve/delve/pkg/dwarf/line
#18 6.486 github.com/mattn/go-runewidth
#18 6.488 go.starlark.net/syntax
#18 6.589 github.com/peterh/liner
#18 6.596 github.com/google/go-dap
#18 6.671 github.com/go-delve/delve/pkg/dwarf/godwarf
#18 6.765 github.com/spf13/pflag
#18 6.818 github.com/russross/blackfriday/v2
#18 6.855 go.starlark.net/resolve
#18 6.925 github.com/go-delve/delve/pkg/dwarf/loclist
#18 6.926 github.com/go-delve/delve/pkg/dwarf/reader
#18 6.970 github.com/go-delve/delve/pkg/config
#18 6.986 github.com/mattn/go-isatty
#18 6.986 github.com/sirupsen/logrus
#18 6.996 go.starlark.net/internal/compile
#18 7.065 github.com/go-delve/delve/pkg/gobuild
#18 7.144 github.com/cpuguy83/go-md2man/v2/md2man
#18 7.149 go.starlark.net/starlark
#18 7.157 github.com/spf13/cobra
#18 7.159 github.com/go-delve/delve/pkg/logflags
#18 7.193 github.com/go-delve/delve/pkg/proc
#18 7.296 github.com/spf13/cobra/doc
#18 7.644 github.com/go-delve/delve/service/api
#18 7.644 github.com/go-delve/delve/pkg/proc/winutil
#18 7.644 github.com/go-delve/delve/pkg/proc/amd64util
#18 7.665 github.com/go-delve/delve/pkg/proc/core/minidump
#18 7.680 github.com/go-delve/delve/pkg/proc/linutil
#18 7.709 github.com/go-delve/delve/pkg/locspec
#18 7.734 github.com/go-delve/delve/pkg/proc/core
#18 7.734 github.com/go-delve/delve/pkg/proc/gdbserial
#18 7.734 github.com/go-delve/delve/pkg/proc/native
#18 7.896 github.com/go-delve/delve/service/debugger
#18 7.996 github.com/go-delve/delve/service
#18 8.023 github.com/go-delve/delve/service/rpc1
#18 8.023 github.com/go-delve/delve/service/rpc2
#18 8.097 github.com/go-delve/delve/service/rpccommon
#18 8.098 github.com/go-delve/delve/pkg/terminal/starbind
#18 8.233 github.com/go-delve/delve/pkg/terminal
#18 8.386 github.com/go-delve/delve/service/dap
#18 8.503 github.com/go-delve/delve/cmd/dlv/cmds
#18 8.561 github.com/go-delve/delve/cmd/dlv
#18 9.587 go: downloading github.com/mdempsky/gocode v0.0.0-20200405233807-4acdcbdea79d
#18 10.01 go: downloading golang.org/x/tools v0.1.2
#18 10.91 github.com/mdempsky/gocode/internal/lookdot
#18 10.91 golang.org/x/tools/go/internal/gcimporter
#18 10.93 github.com/mdempsky/gocode/internal/suggest
#18 11.10 golang.org/x/tools/go/gcexportdata
#18 11.12 github.com/mdempsky/gocode/internal/cache
#18 11.14 github.com/mdempsky/gocode/internal/gbimporter
#18 11.16 github.com/mdempsky/gocode
#18 11.67 go: downloading github.com/uudashr/gopkgs v1.3.2
#18 11.74 go: downloading github.com/uudashr/gopkgs/v2 v2.1.2
#18 11.77 go: downloading github.com/uudashr/gopkgs v2.0.1+incompatible
#18 11.82 go: downloading github.com/karrick/godirwalk v1.12.0
#18 11.82 go: downloading github.com/pkg/errors v0.8.1
#18 11.97 go: downloading github.com/pkg/errors v0.9.1
#18 11.97 go: downloading github.com/karrick/godirwalk v1.16.1
#18 12.08 github.com/pkg/errors
#18 12.08 github.com/karrick/godirwalk
#18 12.13 github.com/uudashr/gopkgs/v2
#18 12.15 github.com/uudashr/gopkgs/v2/cmd/gopkgs
#18 12.55 go: downloading github.com/ramya-rao-a/go-outline v0.0.0-20200117021646-2a048b4510eb
#18 12.76 golang.org/x/tools/go/buildutil
#18 12.80 github.com/ramya-rao-a/go-outline
#18 13.21 go: downloading github.com/acroca/go-symbols v0.1.1
#18 13.39 github.com/acroca/go-symbols
#18 13.70 go: downloading golang.org/x/sys v0.0.0-20210510120138-977fb7262007
#18 13.70 go: downloading golang.org/x/mod v0.4.2
#18 13.70 go: downloading golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1
#18 14.13 golang.org/x/tools/cmd/guru/serial
#18 14.13 golang.org/x/mod/semver
#18 14.13 golang.org/x/xerrors/internal
#18 14.15 golang.org/x/tools/go/ast/astutil
#18 14.15 golang.org/x/sys/execabs
#18 14.15 golang.org/x/tools/internal/event/label
#18 14.16 golang.org/x/xerrors
#18 14.16 golang.org/x/tools/container/intsets
#18 14.16 golang.org/x/tools/internal/fastwalk
#18 14.16 golang.org/x/tools/internal/typesinternal
#18 14.16 golang.org/x/tools/refactor/importgraph
#18 14.18 golang.org/x/tools/go/internal/cgo
#18 14.18 golang.org/x/tools/internal/event/keys
#18 14.22 golang.org/x/mod/module
#18 14.22 golang.org/x/tools/internal/gopathwalk
#18 14.29 golang.org/x/tools/go/types/typeutil
#18 14.29 golang.org/x/tools/go/loader
#18 14.30 golang.org/x/tools/internal/event/core
#18 14.33 golang.org/x/tools/go/ssa
#18 14.33 golang.org/x/tools/internal/event
#18 14.36 golang.org/x/tools/internal/gocommand
#18 14.39 golang.org/x/tools/go/internal/packagesdriver
#18 14.39 golang.org/x/tools/internal/packagesinternal
#18 14.39 golang.org/x/tools/internal/imports
#18 14.41 golang.org/x/tools/go/packages
#18 14.66 golang.org/x/tools/imports
#18 14.74 golang.org/x/tools/go/callgraph
#18 14.74 golang.org/x/tools/go/ssa/ssautil
#18 14.77 golang.org/x/tools/go/pointer
#18 14.77 golang.org/x/tools/go/callgraph/static
#18 15.02 golang.org/x/tools/cmd/guru
#18 15.74 golang.org/x/tools/refactor/satisfy
#18 15.80 golang.org/x/tools/refactor/rename
#18 15.91 golang.org/x/tools/cmd/gorename
#18 16.28 go: downloading github.com/cweill/gotests v1.6.0
#18 16.37 go: downloading golang.org/x/tools v0.0.0-20191109212701-97ad0ed33101
#18 17.52 github.com/cweill/gotests/internal/models
#18 17.54 github.com/cweill/gotests/internal/render/bindata
#18 17.54 github.com/cweill/gotests/templates
#18 17.56 github.com/cweill/gotests/internal/goparser
#18 17.56 github.com/cweill/gotests/internal/input
#18 17.59 github.com/cweill/gotests/internal/render
#18 17.61 github.com/cweill/gotests/internal/output
#18 17.63 github.com/cweill/gotests
#18 17.65 github.com/cweill/gotests/gotests/process
#18 17.66 github.com/cweill/gotests/gotests
#18 18.16 go: downloading github.com/fatih/gomodifytags v1.13.0
#18 18.28 go: downloading github.com/fatih/structtag v1.2.0
#18 18.28 go: downloading github.com/fatih/camelcase v1.0.0
#18 18.28 go: downloading golang.org/x/tools v0.0.0-20180824175216-6c1c5e93cdc1
#18 18.73 github.com/fatih/camelcase
#18 18.75 github.com/fatih/structtag
#18 18.76 github.com/fatih/gomodifytags
#18 19.07 go: downloading github.com/josharian/impl v1.0.0
#18 19.25 go: downloading golang.org/x/tools v0.0.0-20200522201501-cb1345f3a375
#18 19.73 go: downloading golang.org/x/mod v0.2.0
#18 19.78 go: downloading golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
#18 19.95 github.com/josharian/impl
#18 20.38 go: downloading github.com/davidrjenni/reftools v0.0.0-20210213085015-40322ffdc2e4
#18 20.43 go: downloading golang.org/x/tools v0.0.0-20190408220357-e5b8258f4918
#18 21.16 github.com/davidrjenni/reftools/cmd/fillstruct
#18 21.63 go: downloading github.com/haya14busa/goplay v1.0.0
#18 21.78 go: downloading github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
#18 21.91 github.com/skratchdot/open-golang/open
#18 21.91 github.com/haya14busa/goplay
#18 21.94 github.com/haya14busa/goplay/cmd/goplay
#18 22.56 go: downloading github.com/godoctor/godoctor v0.0.0-20200702010311-8433dcb3dc61
#18 22.92 go: downloading golang.org/x/tools v0.0.0-20191028215554-80f3f9ca0853
#18 22.92 go: downloading github.com/willf/bitset v1.1.10
#18 23.05 go: downloading github.com/willf/bitset v1.2.0
#18 23.54 go get: github.com/willf/bitset@v1.1.10 updating to
#18 23.54       github.com/willf/bitset@v1.2.0: parsing go.mod:
#18 23.54       module declares its path as: github.com/bits-and-blooms/bitset
#18 23.54               but was required as: github.com/willf/bitset
#18 ERROR: executor failed running [/bin/sh -c go get -v -u github.com/go-delve/delve/cmd/dlv     && go get -v -u github.com/mdempsky/gocode     && go get -v -u github.com/uudashr/gopkgs/v2/cmd/gopkgs     && go get -v -u github.com/ramya-rao-a/go-outline     && go get -v -u github.com/acroca/go-symbols     && go get -v -u golang.org/x/tools/cmd/guru     && go get -v -u golang.org/x/tools/cmd/gorename     && go get -v -u github.com/cweill/gotests/...     && go get -v -u github.com/fatih/gomodifytags     && go get -v -u github.com/josharian/impl     && go get -v -u github.com/davidrjenni/reftools/cmd/fillstruct     && go get -v -u github.com/haya14busa/goplay/cmd/goplay     && go get -v -u github.com/godoctor/godoctor     && go get -v -u github.com/rogpeppe/godef     && go get -v -u github.com/sqs/goreturns     && go get -v -u golang.org/x/lint/golint     && go get -v -u github.com/zmb3/gogetdoc     && go get -v -u honnef.co/go/tools/...     && go get -v -u golang.org/x/tools/cmd/goimports     && go get -v -u github.com/mgechev/revive]: exit code: 1
```

**Comment by CelsoSantos** [comment]
I confirm the same scenario on my end

**Comment by wassimkha** [comment]
@MarkDHarris @CelsoSantos did you guys find a fix for this?

**Comment by CelsoSantos** [comment]
In my case I ended up commenting out that line. In the meantime I migrated to MS provided images

**Comment by rdallman** [comment]
fixed #57
